pandas/core/frame.py

```diff
@@ -8801,6 +8804,18 @@ def isin(self, values) -> "DataFrame":
         axis=0, doc="The column labels of the DataFrame."
     )
 
+    @property
+    def _AXIS_NUMBERS(self) -> Dict[str, int]:
+        """.. deprecated:: 1.1.0"""
+        super()._AXIS_NUMBERS
+        return {"index": 0, "columns": 1}
+
+    @property
+    def _AXIS_NAMES(self) -> Dict[int, str]:
+        """.. deprecated:: 1.1.0"""
+        super()._AXIS_NAMES
+        return {0: "index", 1: "columns"}
+
```

Comment:
Both  ``_AXIS_NUMBERS`` and ``_AXIS_NAMES`` are not needed anymore. Just in case they are used downstream I've deprecated them instead of just removing them.